pubspec.yaml

```diff
@@ -13,7 +13,7 @@ description: A mobile application for students at ACS UPB.
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 #
 # ACS UPB Mobile uses semantic versioning. You can read more in the CONTRIBUTING.md file.
-version: 1.3.1+42
+version: 1.3.2
```

Comment:
You can't just remove the build number, you need to increment it as well (even if you also increment the minor version). The bot is showing an error about this as well, please don't ignore.

Reply:
Incremented it to +1

Reply:
maybe a personal suggestion of how I could increment this, I declare myself a dummy